app/assets/javascripts/listings/ListingService.js.coffee

```diff
@@ -347,6 +365,9 @@ ListingService = ($http, $localStorage, $modal, $q, $state) ->
   Service.listingIs = (name) ->
     Service.LISTING_MAP[Service.listing.Id] == name
 
+  Service._returnPreferenceName = (preference) ->
```

Comment:
@chrisdolendo does really this need to be pulled out into its own function, if it's really just for:

```coffee
preferenceNames = _.map Service.listing.preferences, (pref) -> pref.preferenceName
```